app/assets/javascripts/app/views/item_view.coffee

```diff
@@ -1,6 +1,63 @@
 class App.views.ItemView extends Backbone.View
-  initialize: (item) ->
-    @item = item
+  className: 'item'
+  tagName: 'li'
+  template: HandlebarsTemplates['item']
+
+  initialize: (options) ->
+    @item = options.item
+    @children = new App.collections.Items(@item.get('children'))
+    @$el.attr('id', @item.id)
+    @showChildren = true
+
+  events:
+    'click .toggle-children': 'toggleChildren'
+    'click h2': 'toggleh2Editable'
+    'click p': 'togglepEditable'
+
+  toggleh2Editable: (e) ->
```

Comment:
Using implementation details like `h2` and `p` in method and variable names is not good practice.
What happens if we change the heading to H3? or if we change p to div in the markup?

A better way to do this is to define a semantically descriptive class for the DOM element, like `item-title` and `item-body`